platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/CertificateGenerator.java

```diff
@@ -58,6 +60,7 @@ public KeyStoreFile getKeyStoreFile() {
     return CertificateCommand.getKeyStoreFile();
   }
 
+  @SuppressWarnings("squid:S2583") /* objectName is not always non-null, see below */
```

Comment:
❗️ Incorrect suppression; remove.

For the `mBeanServer` to be non-null on line 75, you'd have successfully created the `objectName`.